src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.bigSearch {
+  display: block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  margin: 20px 0;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  height: 70px;
+  background-color: #fff;
+  box-sizing: border-box;
```

Comment:
you can set `box-sizing: border-box;` in `*`. That would help to avoid duplicating